README.md

```diff
@@ -3,60 +3,70 @@
+The `galasaVersion` value is the version of Galasa you want to run. You should not use latest to ensure each pod in the Ecosystem is running at the same level.
 
-The Galasa Ecosystem Helm chart will deploy three persistent volumes.  If you need to provide a Kubernetes storage class for these PVs, then please use the following command:-
+The `externalHostname` value is the DNS hostname or IP address of the Kubernetes node that be used to access the Galasa NodePort services. If you are deploying to minikube, this can be retrieved by running `minikube ip`.
 
-    helm install --set storageClass=mysgtorageclass --set galasaVersion=0.23.0 --set externalHostname=my.host.name <release-name> galasa/ecosystem --wait 
+After the Ecosystem has been successfully deployed, you can run the following command to ensure the Ecosystem can be accessed externally to Kubernetes and a simple test engine can be run:
 
-After the Ecosystem has been successfully deployed, you can run the following command to ensure the Ecosystem can be accessed externally to Kubernetes and a simple test engine can be run:-
+```
+helm test <release-name>
+```
 
-    helm test <release-name>
+Once the Ecosystem is up and running, you will need to determine the URL of the Ecosystem bootstrap. Issue the command:
 
-Once the Ecosystem is up and running,  you will need to determine the URL of the Ecosystem bootstrap.  Issue the command:-
+```
+kubectl get svc
+```
 
-    kubectl get svc
-    
-Look for the api-external service and the node port associated with the 8080 port.   Combine that with the external hostname you provided.  for example:-
+Look for the api-external service and the node port associated with the 8080 port.   Combine that with the external hostname you provided. For example:
 
 ```
 test-api-external                 NodePort    10.107.160.208   <none>        9010:31359/TCP,9011:31422/TCP,8080:30960/TCP   18s
 ```
 
-If the external hostname you provided was `example.com`,  the bootstrap URL will be `http://example.com:30960/boostrap`.  You will enter this into the Eclipse plugin preferences, or the galasactl command.
+If the external hostname you provided was `example.com`, the bootstrap URL will be `http://example.com:30960/boostrap`. You will enter this into the Eclipse plugin preferences, or the galasactl command.
 
 ### Upgrading the Galasa Ecosystem chart
 
-If you want to upgrade the Galasa Ecosystem to increase the Galasa version, for example, then you can use the following command:-
+If you want to upgrade the Galasa Ecosystem to increase the Galasa version, for example, then you can use the following command:
+
+```
+helm upgrade --reuse-values --set galasaVersion=0.24.0 --wait
+```
 
-     helm upgrade --reuse-values --set galasaVersion=0.24.0 --wait
-     
 **Warning**: If you have made changes to the configmaps that this helm chart manages, those changes will be lost and will have to be reapplied manually.
```

Comment:
Is there no way to avoid this ? maybe by re-building them each time using values and a template ?

Reply:
Addressed in https://github.com/galasa-dev/helm/pull/1/commits/a4bdb4e4b9da09518e60e029a51f35f36cbaca07